src/include/daos_srv/vos.h

```diff
@@ -425,6 +425,35 @@ vos_obj_fetch(daos_handle_t coh, daos_unit_oid_t oid, daos_epoch_t epoch,
 	      uint64_t flags, daos_key_t *dkey, unsigned int iod_nr,
 	      daos_iod_t *iods, d_sg_list_t *sgls);
 
+/**
+ * Fetch values for the given keys and their indices.
+ * If output buffer is not provided in \a sgl, then this function returns
+ * the directly accessible addresses of record data, upper layer can directly
+ * read from these addresses (rdma mode).
+ *
+ * TODO: add more detail descriptions for punched or missing records.
```

Comment:
Why not do this TODO now?

Reply:
I didn't add this TODO.   I probably need to clean up the vos header at some point but don't want to do it here.